CMakeLists.txt

```diff
@@ -92,13 +92,13 @@ if(BUILD_GAME OR BUILD_EDITOR)
         add_definitions(-DOGRE_PLUGIN_DIR_DBG="${OGRE_PLUGIN_DIR}")
     else()
         find_package(OGRE REQUIRED QUIET)
-        include_directories(${OGRE_INCLUDE_DIRS} ${OGRE_Terrain_INCLUDE_DIRS} ${OGRE_Paging_INCLUDE_DIRS})
+        include_directories(${OGRE_INCLUDE_DIRS} ${OGRE_Terrain_INCLUDE_DIRS} ${OGRE_Paging_INCLUDE_DIRS} ${OGRE_Overlay_INCLUDE_DIRS} ${OGRE_Bites_INCLUDE_DIRS})
```

Comment:
these cmake changes are unnecessary, if Ogre 1.12 is properly installed, it should be taking the branch above.